src/Lykke.Service.HFT.Wamp/Auth/AuthMethods.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Lykke.Service.HFT.Wamp.Auth
+{
+    public static class AuthMethods
+    {
+        public const string Anonymous = "anonymous";
```

Comment:
May be it is better to name it Basic, because it is not anonymous in fact? And in client too.

Reply:
It's a tricky question, because there is no standard "Basic" auth in WAMP. But I think your suggestion is right. I'll experiment with several clients and find the best solution.